sdk/storage/Azure.Storage.Blobs/src/Models/BlobProperties.cs

```diff
@@ -206,8 +206,8 @@ public partial class BlobProperties
         /// <summary>
         /// The tier of page blob on a premium storage account or tier of block blob on blob storage LRS accounts.
         /// For a list of allowed premium page blob tiers, see
-        /// https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage#features. For blob
-        /// storage LRS accounts, valid values are Hot/Cool/Archive.
+        /// https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage#features. For general
+        /// purpose v2 and blob storage account types, the valid values are Hot/Cool/Archive.
```

Comment:
```suggestion
        /// <see href="https://docs.microsoft.com/azure/virtual-machines/windows/premium-storage#features" />. For general
        /// purpose v2 and blob storage account types, the valid values are:
        /// <list>
        ///  <item><description>Hot</description></item>
        ///  <item><description>Cool</description></item>
        ///  <item><description>Archive</description></item>
        /// </list>
```

Reply:
If nothing else, please be sure to strip the language slug (`en/us`) from the URL.